### PR TITLE
feat: implement event additional options

### PR DESCRIPTION
**Pull request by ivoreis**
This PR implements some additional options for DD events, as per https://docs.datadoghq.com/developers/dogstatsd/datagram_shell/?tab=events

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,6 +76,14 @@ client.service_check("redis.can_connect", ServiceStatus::OK, tags, Some(service_
 
 // Send a custom event
 client.event("My Custom Event Title", "My Custom Event Body", tags).unwrap();
+
+// Send a custom event with options - https://docs.datadoghq.com/developers/dogstatsd/datagram_shell/?tab=events
+let event_options = EventOptions::new()
+    .with_timestamp(1638480000)
+    .with_hostname("localhost")
+    .with_priority("normal")
```
Comment (ivoreis) [inline README.md]: The examples need to be updated to use enums

**Comment by mcasper** [comment]
Hey @ivoreis, thanks so much for the contribution, it looks great! If you can update the README examples to match the API (use the enum types) I think we'll be good to go!

**Comment by ivoreis** [comment]
Done 👍

**Comment by mcasper** [comment]
Thanks @ivoreis ! This has been released in 0.12.0

**Pull request merged**